dali/sasha/saserver.cpp

```diff
@@ -238,6 +240,29 @@ class CSashaCmdThread : public CInterface, implements IPooledThread
 
 void SashaMain()
 {
+    if (!isDaliClient)
+    {
+        bool stopped = false;
+        class AbortHandler : implements CSimpleInterfaceOf<IAbortHandler>
+        {
+            bool stopped = false;
+        public:
+            AbortHandler(bool &_stopped) : stopped(_stopped) {}
+            virtual bool onAbort() override
+            {
+                PROGLOG("Aborted");
+                stopped = true;
```

Comment:
This isn't going to affect the 'stopped' outside of the class, so it won't cause it to stop at the moment.
this.stopped needs to be a reference to the stopped passed in.